conf/settings.py

```
@@ -574,3 +574,16 @@
     'CONTACT_INDUSTRY_USER_REPLY_TO_ID',
     None
 )
+
+CAPITAL_INVEST_AGENT_TEMPLATE_ID = env.str(
+    'CAPITAL_INVEST_AGENT_TEMPLATE_ID',
+    '7d6e2bd5-f6a5-4050-9302-31bd033a2d8b'
+)
+CAPITAL_INVEST_USER_TEMPLATE_ID = env.str(
+    'CAPITAL_INVEST_USER_TEMPLATE_ID',
+    '11cef6bd-c3e6-43c6-8959-f749f3991188'
+)
+CAPITAL_INVEST_USER_REPLY_TO_EMAIL = env.str(
+    'CAPITAL_INVEST_USER_REPLY_TO_EMAIL',
+    None
```

this can hide errors in prod: if we forget that vault value then code that needs `CAPITAL_INVEST_USER_REPLY_TO_EMAIL` would fail when the code is executed.

please do `CAPITAL_INVEST_USER_REPLY_TO_EMAIL = env.str('CAPITAL_INVEST_USER_REPLY_TO_EMAIL')` so it fails on app start (therefore deployment fails and rollback is triggerered so the user is not affected).

in makefile test env vars we can use placeholder values

---

Ok, the `CONTACT_INDUSTRY_USER_REPLY_TO_ID` var above this does the same thing, should that be changed too? Also, in gov notify there is a default reply-to email which I believe is used when this is none and so doens't error I don't think

---

Yeah I don't have it in my makefile and it works fine and just defaults to the default reply-to email from gov notify